src/components/HelpResourcesCards/HelpResourcesCards.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+import { Link as RouterLink } from "react-router-dom";
+
+/*Import Icons from https://designsystem.digital.gov/components/icon/ using title case*/
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  IconHelpOutline,
+  IconReduceCapacity,
+  IconFlag,
+  IconMap,
+  IconAlternateEmail,
+  CardGroup,
+} from "@trussworks/react-uswds";
+
+export const HelpResourcesCards = () => {
+  return (
+    <aside>
+      <CardGroup>
+        <Card gridLayout={{ tablet: { col: 4 } }}>
+          <CardHeader>
+            <h2>Related Resources</h2>
+            <IconReduceCapacity size={4} title="Icon for API Docs" />
+          </CardHeader>
+
+          <CardBody>
+            <p>View resources related to CAMD data</p>
+          </CardBody>
+
+          <CardFooter>
+            <RouterLink to="/related-resources">Related Resources</RouterLink>
+          </CardFooter>
+        </Card>
+
+        <Card gridLayout={{ tablet: { col: 4 } }}>
+          <CardHeader align="center">
+            <h2>Release Notes</h2>
+            <IconFlag
+              size={4}
+              title="Icon for other card"
+              className="palette-color-grayscale"
+            />
+          </CardHeader>
+
+          <CardBody>
+            <p>Information on CAM API Updates.</p>
+          </CardBody>
+
+          <CardFooter>
+            <RouterLink to="/release-notes">Release Notes</RouterLink>
+          </CardFooter>
+        </Card>
+
+        <Card gridLayout={{ tablet: { col: 4 } }}>
+          <CardHeader align="center">
+            <h2>FAQs</h2>
+            <IconHelpOutline
+              size={4}
+              title="Icon for other card"
+              className="palette-color-grayscale"
+            />
+          </CardHeader>
+
+          <CardBody>
+            <p>Frequently Asked Questions about the CAM API.</p>
+          </CardBody>
+
+          <CardFooter>
+            <RouterLink to="/faqs">FAQs</RouterLink>
+          </CardFooter>
+        </Card>
+
+        <Card gridLayout={{ tablet: { col: 4 } }}>
+          <CardHeader align="center">
+            <h2>API Guides</h2>
```

Comment:
We probably won't have this page ready by the time we release, so maybe add a display element to make this not visible?